backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/AESCryptoService.java

```diff
@@ -0,0 +1,88 @@
+package com.canvas.service.helperServices;
+
+import com.canvas.exceptions.CanvasAPIException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.Base64;
+
+/**
+ * AESCryptoService class that provides methods to encrypt and decrypt Access tokens.
+ */
+@Component
+public class AESCryptoService {
+    private static SecretKeySpec secretKey;
+
+    /**
+     * Constructor that sets the secret key using the environment property "canvas.secretKey".
+     *
+     * @param env environment instance
+     */
+    @Autowired
+    public AESCryptoService(Environment env) {
+        this.setKey(env.getProperty("canvas.secretKey"));
```

Comment:
This will never be null because we are reading it from the applications.properties file. Our application shouldn't run without the client secret from the applications.properties file.